participants-schema.json

```
@@ -0,0 +1,47 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Participants Schema",
+  "type": "object",
+  "definitions": {
+    "url": {
+      "type": "string",
+      "pattern": "^(http|https)://[A-Za-z0-9ÄÖÜäöüß\\-_\\./]+"
```

```suggestion
      "pattern": "^https?://\S+$"
```

Don't forget all the other alphabets.

---

I'm not sure, if that works, as regex in JSON Schema are limited. At least, we need to escape the backslash

---

it works :)